Binary files orig/rgdal/src/.Makevars.ucrt.swp and patched/rgdal/src/.Makevars.ucrt.swp differ
diff -Nru orig/rgdal/src/Makevars.ucrt patched/rgdal/src/Makevars.ucrt
--- orig/rgdal/src/Makevars.ucrt	2022-05-07 14:06:06.000000000 +0200
+++ patched/rgdal/src/Makevars.ucrt	2022-05-31 15:47:28.277608100 +0200
@@ -3,7 +3,7 @@
 	-DPROJ_H_API
 
 PKG_LIBS = \
-        -lgdal -lgta -lsqlite3 -lmysqlclient -lspatialite -lproj -lgeos_c -lgeos  \
+        -lgdal -lkea -lhdf5_cpp -lhdf5 -lgta -lsqlite3 -lmysqlclient -lspatialite -lproj -lgeos_c -lgeos  \
         -ljson-c -lnetcdf -lpq \
         -lpoppler -llcms2 -lfreetype -lharfbuzz -lfreetype -llz4 -lpcre2-8 \
         -lwebp -lcurl -lidn2 -lunistring -lssh2 -lgcrypt -lgpg-error -lssl \
